### ISSUE TITLE
[TCK TESTING FRAME]Cannot parse column contains "__NULL__"

### ISSUE THREAD

**Issue by Aiee** [vesoft-inc/nebula-graph#614]
```
Query: 
      """
      MATCH (v) -[*0..1]-(v2:player{name: "Tim Duncan"})-[e*1..2]->(v3)
      RETURN DISTINCT v3.name AS Name,v3.age AS Age ORDER BY Age limit 10
      """


Console Result:
    +---------------------+----------+
    | Name                | Age      |
    +---------------------+----------+
    | "Kyle Anderson"     | 25       |
    +---------------------+----------+
    | "Danny Green"       | 31       |
    +---------------------+----------+
    | "Marco Belinelli"   | 32       |
    +---------------------+----------+
    | "LaMarcus Aldridge" | 33       |
    +---------------------+----------+
    | "LeBron James"      | 34       |
    +---------------------+----------+
    | "Tony Parker"       | 36       |
    +---------------------+----------+
    | "Manu Ginobili"     | 41       |
    +---------------------+----------+
    | "Tim Duncan"        | 42       |
    +---------------------+----------+
    | "Spurs"             | __NULL__ |
    +---------------------+----------+
    | "Raptors"           | __NULL__ |
    +---------------------+----------+


TCK error:
========================================================================= FAILURES ==========================================================================
______________________________________________________ test_multiple_variable_length_with_edge_filter _______________________________________________________
../../../../.local/lib/python3.7/site-packages/pytest_bdd/scenario.py:179: in scenario_wrapper
    _execute_scenario(feature, scenario, request, encoding)
../../../../.local/lib/python3.7/site-packages/pytest_bdd/scenario.py:145: in _execute_scenario
    _execute_step_function(request, scenario, step, step_func)
../../../../.local/lib/python3.7/site-packages/pytest_bdd/scenario.py:115: in _execute_step_function
    return_value = step_func(**kwargs)
../../tests/tck/conftest.py:248: in result_should_be
    cmp_dataset(request, graph_spaces, result, order=False, strict=True)
../../tests/tck/conftest.py:178: in cmp_dataset
    graph_spaces.get("variables", {}),
../../tests/tck/utils/table.py:51: in dataset
    ]) for row in string_table['rows']
../../tests/tck/utils/table.py:51: in <listcomp>
    ]) for row in string_table['rows']
../../tests/tck/utils/table.py:50: in <listcomp>
    _parse_value(row[i], variables) for i in range(len(column_names))
../../tests/tck/utils/table.py:25: in _parse_value
    assert value is not None, f"parse error: column is {cell}"
E   AssertionError: parse error: column is __NULL__
```

**Comment by jyz0309** [comment]
Do the issue is a high-priority? Can I try it?

**Comment by HarrisChu** [comment]
fixed in v2.0.1
